packages/victory-bar/src/index.d.ts

```diff
@@ -19,7 +19,6 @@ export interface VictoryBarProps
     VictoryDatableProps,
     VictoryMultiLabelableProps {
   alignment?: VictoryBarAlignmentType;
-  ariaLabel?: StringOrCallback;
```

Comment:
Removed these types from `VictoryBar`, although the props if provided will still apply, just gives a ts error in the demo. There doesn't appear to be a way to prevent the ariaLabel from applying to `VictoryBar` separate from `Bar`